source/jormungandr/tests/here_distributed_routing_tests.py

```diff
@@ -41,6 +41,7 @@
                 "args": {
                     "apiKey": "bob_id",
                     "service_base_url": "route.bob.here.com/routing/7.2/",
+                    "language": "french",
```

Comment:
Is it possible to test the language = 'french' from this configuration?

Reply:
Nope. The response test is mocked. https://github.com/CanalTP/navitia/pull/3244/files/98d4bf18d1ff21949b37157dcdb33c079584f276#diff-9e76c2ecb87559065484a2eed06cf773R93

Reply:
I added it, to have a complete example of configuration for beginners